dts/arm/nordic/nrf5340_cpuapp.dtsi

```diff
@@ -85,7 +84,7 @@
 			/* Common nRF5340 Application MCU
 			 * peripheral description
 			 */
-			#include "nrf5340_cpuapp_common.dtsi"
+			#include "nrf5340_cpuapp_peripherals.dtsi"
```

Comment:
Some of those peripherals (like WDT1, PWM3,  SPIM/SPIS/TWIM/TWIS/UARTE2-3) are not functional in the Eng A revision. I think we should delete their nodes in `*_eng_a.dtsi` files, to prevent them from being used.

Reply:
Agreed, @anangl , is this the ones I know about? I 'll add a separate commit and you can review it :)

Reply:
@anangl could you please provide @ioannisg with a list?

Reply:
> Could we change UART DTS pin configuration for Network Core within this PR?
> See:
> [#26110 (comment)](https://github.com/zephyrproject-rtos/zephyr/pull/26110#discussion_r444717827)
> for details

done, pls check if needed.

Reply:
applied now @anangl , on a separate commit on top.
Please note that I chose to "manually" delete the nodes in the ENG_A .dts file, instead of providing 2 file versions for nrf5340_cpuapp_peripherals, one for eng_a and one for fp1. I suppose this is an accepted way to do this.